apps/api/src/app/controllers/__tests__/__snapshots__/fetchManyViewModels.e2e.spec.ts.snap

```diff
@@ -609,7 +609,7 @@ Object {
         },
       ],
       "data": Object {
-        "contributorAndRoles": Array [
+        "contributions": Array [
```

Comment:
This snapshot failure is do to a break in the API contract. I have notified Justin who has a stand-alone project that uses this part of the API.